e2e/atlas/datalake_test.go

```
@@ -120,17 +107,35 @@ func TestDatalake(t *testing.T) {
 		resp, err := cmd.CombinedOutput()
 
 		if err != nil {
-			t.Fatalf("unexpected error: %v, resp: %v", err, string(resp))
+			t.Fatalf("unexpected error: %v, resp: %v\n", err, string(resp))
 		}
 
-		datalake := atlas.DataLake{}
-		err = json.Unmarshal(resp, &datalake)
-		if err != nil {
+		var datalake atlas.DataLake
+		if err = json.Unmarshal(resp, &datalake); err != nil {
 			t.Fatalf("unexpected error: %v", err)
 		}
 
-		if err := deep.Equal(datalake.DataProcessRegion.Region, updateRegion); err != nil {
-			t.Error(err)
+		if datalake.DataProcessRegion.Region != updateRegion {
+			t.Errorf("got=%#v\nwant=%#v\n", datalake.DataProcessRegion.Region, updateRegion)
+		}
+	})
+
+	t.Run("Delete", func(t *testing.T) {
+		cmd := exec.Command(cliPath,
+			atlasEntity,
+			datalakeEntity,
+			"delete",
+			datalakeName,
+			"--force")
+		cmd.Env = os.Environ()
+
+		resp, err := cmd.CombinedOutput()
+		if err != nil {
+			t.Fatalf("unexpected error: %v, resp: %v\n", err, string(resp))
+		}
+		expected := fmt.Sprintf("Project whitelist entry '%s' deleted\n", datalakeName)
+		if string(resp) != expected {
+			t.Errorf("got=%#v\nwant=%#v\n", string(resp), expected)
```

adding this to make sure this does not silently fail next